pkg/connector/namespaces.go

```diff
@@ -222,7 +213,8 @@ func (o *namespaceBuilder) Revoke(ctx context.Context, g *v2.Grant) (annotations
 	spec := user.GetSpec()
 	_, ok := spec.GetAccess().GetNamespaceAccesses()[namespaceID]
 	if !ok {
-		return nil, fmt.Errorf("temporalcloud-connector: grant does not exist for user")
+		annos := annotations.New(&v2.GrantAlreadyRevoked{})
+		return annos, fmt.Errorf("temporalcloud-connector: grant does not exist for user")
```

Comment:
If it's already revoked, we usually don't error (since that would be a provisioning error in c1). Same for grant already exists above.